### DISCUSSION TITLE
How to control Privilege

### DISCUSSION THREAD

**Discussion by HRNasr** [microsoft/ebpf-for-windows#2517] [Q&A] [answered]
hello
current eBPF for windows can use for develop program  with target of    application control and file access?
Allow/Denied run a application ?
Allow/Black access to a file?

**Comment by dahavey** [comment, marked as answer]
We don't have those hooks in Windows right now. They are supported in Linux: BPF LSM, Kprobes. https://docs.kernel.org/bpf/prog_lsm.html
